data/json/items/gun/10mm.json

```
@@ -44,5 +44,30 @@
     "barrel_length": "250 ml",
     "magazine_well": "150 ml",
     "magazines": [ [ "10mm", [ "glock20mag", "glock20bigmag", "glock20drum50rd" ] ] ]
+  },
+  {
+    "id": "kriss_vector10",
+    "copy-from": "smg_base",
+    "type": "GUN",
+    "looks_like": "TDI",
+    "name": { "str": "Kriss Vector 10mm Auto" },
+    "description": "A submachine gun with a sleek in-line design. It features a 2-round burst mode to make the least out of its unique recoil mitigation system alongside a blistering full-auto mode. This one is chambered in 10mm Auto and uses Glock 20 magazines.",
```

Double spaces rule in description text. Also, "make the least out of" sounds incorrect.
```suggestion
    "description": "A submachine gun with a sleek in-line design.  It features a 2-round burst mode to make the most out of its unique recoil mitigation system alongside a blistering full-auto mode.  This one is chambered in 10mm Auto and uses Glock 20 magazines.",
```